docs/resource-schemas.md

```diff
@@ -22,6 +22,15 @@ library:
       name: Storybook
       action: link
       url: https://react.carbondesignsystem.com
+  designKits:
+    carbon-white-sketch:
+      $ref: https://unpkg.com/@carbon-platform/resources/carbon.yml#/designKits/carbon-white-sketch
+    carbon-g10-sketch:
+      $ref: https://unpkg.com/@carbon-platform/resources/carbon.yml#/designKits/carbon-g10-sketch
+    carbon-g90-sketch:
+      $ref: https://unpkg.com/@carbon-platform/resources/carbon.yml#/designKits/carbon-g90-sketch
+    carbon-g100-sketch:
+      $ref: https://unpkg.com/@carbon-platform/resources/carbon.yml#/designKits/carbon-g100-sketch
```

Comment:
should we also add a top level `designKits` here ?

Reply:
Done. I think we'll want to clean up these examples when we move the resource-schemas.md to carbondesignsystem.com/contributing/schema, or whatever that URL will be.